runtime/themes/solarized_dark.toml

```diff
@@ -73,18 +73,20 @@
 "error" = { fg = "red", modifiers= ["bold", "underlined"] }
 "info" = { fg = "blue", modifiers= ["bold", "underlined"] }
 "hint" = { fg = "base01", modifiers= ["bold", "underlined"] }
-"diagnostic" = { mdifiers = ["underlined"] }
+"diagnostic" = { modifiers = ["underlined"] }
 
 [palette]
 # 深色 越来越深
-base03  = "#002b36"
-base02  = "#073642"
-base01  = "#586e75"
-base00  = "#657b83"
-base0   = "#839496"
-base1   = "#93a1a1"
-base2   = "#eee8d5"
-base3   = "#fdf6e3"
+base03   = "#002b36"
+base02   = "#073642"
+base0175 = "#16404b"
+base015  = "#2c4f59"
```

Comment:
I don't think base16 have these colors, do you think it would be better to leave some comments here to let future readers know what happen here?

Reply:
Under the values section: https://ethanschoonover.com/solarized/

I think `015`/`0175` are colors between 01 and 02